python/src/scipp/plotting/resampling_model.py

```diff
@@ -120,22 +123,32 @@ def _call_resample(self):
             self._home = self._resampled
             self._home_params = self._resampled_params
 
+    def update_array(self, array):
+        """
+        Update the internal array with a new array.
+        """
+        self._array = array
```

Comment:
Should maybe rename the `__init__` arg `data` to `array` for consistency? Also, maybe make `array` a property instead?

Reply:
i would say change the argument to the init, but not sure about the property name, as we are not really returning the array, we are returning `self._resampled`

Reply:
Well, that is another property?